ctapipe/reco/sklearn.py

```diff
@@ -117,6 +120,10 @@ def __init__(self, subarray=None, models=None, **kwargs):
                     "__init__() missing 1 required positional argument: 'subarray'"
                 )
 
+            if self.prefix is None:
+                # Default prefix is model_cls
+                self.prefix = self.model_cls
```

Comment:
You should either mention that the default is the model class in the help string above, or you might even be able to use a `@default` decorator to link them

Reply:
I am not sure how to use the decorator for this. I would have to make a separate function which sets the prefix, right? I think mentioning the default in the help string is the easier solution in that case.